build

```diff
@@ -0,0 +1,18 @@
+<Project>
+    <ItemGroup>
+        <VersionFile Include="version"/>
+    </ItemGroup>
+    <!-- <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" /> -->
+    <PropertyGroup>
+        <!-- <PackageVersion>$(PackageVersion)</PackageVersion> -->
+        <NuspecPath>"Microsoft.jQuery.Unobtrusive.Validation.nuspec"</NuspecPath>
+    </PropertyGroup>
+    <Target Name ="Build">
+        <ReadLinesFromFile File="@(VersionFile)">
+            <Output TaskParameter="Lines" PropertyName="PackageVersion"/>
+        </ReadLinesFromFile>
+        <Exec Command="npm version --no-git-tag-version --allow-same-version $(PackageVersion)" />
```

Comment:
Just checking - does this change the `package.json` file on disk? If so, I'm concerned that someone will check in the updated version by mistake.

Reply:
If the "version" file is updated it picks up that version or stays the same.